pkg/testutil/util.go

```diff
@@ -350,3 +352,26 @@ func UploadSchema2Image(ctx context.Context, repo distribution.Repository, tag s
 
 	return manifest, nil
 }
+
+func ConvertImage(image *imageapi.Image) (*imageapiv1.Image, error) {
```

Comment:
this whole function is interesting.  tries to replicate what the schema conversion would have done, at least sufficient for our tests.